server/namespace/classifer.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package namespace
+
+import "github.com/pingcap/pd/server/core"
+
+// DefaultNamespace is the namespace all the store and region belong to by
+// default.
+const DefaultNamespace = "global"
+
+// DefaultClassifer is a classifier that classifies all regions and stores to
+// DefaultNamespace.
+var DefaultClassifer = defaultClassifer{}
```

Comment:
I think we need a mutex for protecting the `DefaultClassifer`, since we allow users to override it. And need to export an API to do so

Reply:
We won't allow users to change it as @siddontang suggested. The `var` is due to grammatical limitations.